### PR TITLE
Create new nodes at the correct position.

### PR DESCRIPTION
**Pull request by MichaelMauderer**
### Pull Request Description
When pressing `enter` nodes area created at the wrong position (and the searcher appears at the wrong node position).
This PR fixes that by using the correct position for node creation. 

### Checklist
Please include the following checklist in your PR:

- [x] The documentation has been updated if necessary.
- [x] All code conforms to the [Rust](https://github.com/luna/enso/blob/main/docs/style-guide/rust.md) style guide.
- [x] All code has been tested where possible.
- [x] All code has been profiled where possible.

### CODE REVIEW

File: src/rust/ide/view/graph-editor/src/component/visualization/container.rs [hunk 1]
```diff
@@ -544,6 +544,8 @@ impl Container {
             eval_ frp.enable_fullscreen (model.enable_fullscreen());
             eval_ frp.enable_fullscreen (fullscreen.set_target_value(1.0));
             eval  frp.set_size          ((s) size.set_target_value(*s));
+            trace frp.set_size;
+            trace size.value;
```
Comment (farmaazon) [inline src/rust/ide/view/graph-editor/src/component/visualization/container.rs:548]:
```suggestion
```

File: src/rust/ide/view/graph-editor/src/component/visualization/container.rs [hunk 2]
```diff
@@ -602,6 +604,11 @@ impl Container {
             eval_ on_selected ( action_bar.hide_icons.emit(()) );
         }
 
+        // FIXME[mm]: If we set the size right here, we will see spurious shapes in some
+        // computation heavy circumstances (e.g., collapsing nodes, or creating an new project).
+        // If we let the animation do the size change, the size will only be set after the delays.
+        // This is not optimal the optimal solution to this problem, as it also means that we have
+        // an animation on an invisible component running.
```
Comment (farmaazon) [inline src/rust/ide/view/graph-editor/src/component/visualization/container.rs:611]: This is from another PR I believe? And fix `This is not optimal the optimal solution`

**Review by farmaazon — APPROVED** [review]
It works after brief testing involving much panning and zooming too.

File: src/rust/ide/view/graph-editor/src/component/visualization/container.rs (hunk 2 above)
Comment (MichaelMauderer) [reply src/rust/ide/view/graph-editor/src/component/visualization/container.rs]: Sorry, yes there was a commit from a different PR here. That is gone now.

**Review by wdanilo — APPROVED** [review]

**Pull request merged**